trie/db.go

```diff
@@ -0,0 +1,29 @@
+package trie
+
+//import "sync"
+
+// Database is a high-level wrapper around a db
+// TODO: turn this into an actual database interface
+type Database struct {
+	memDB *MemDB
+}
```

Comment:
We can remove this struct and reference #80

Reply:
okay cool will change that